app/controllers/links_controller.rb

```diff
@@ -0,0 +1,43 @@
+class LinksController < ApplicationController
+  def edit
+    @local_authority = LocalAuthority.find_by(slug: params[:local_authority_slug])
+    @service = Service.find_by(slug: params[:service_slug])
+    @interaction = Interaction.find_by(slug: params[:interaction_slug])
+
+    @link = Link.get_link(params)
+    if @link.nil?
+      @link = Link.new
+      @link.url = 'n/a'
```

Comment:
I think we were happy with not persisting links that don't have a real URL in them.  I think moving the `if nil?` use `n/a` to a helper is a good idea though.